src/api/api.tsx

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios';
+
+import {GoodsData,ThemeData } from '@/types';
```

Comment:
eslint 설정 제대로 되어 있나요 ?

```suggestion
import { GoodsData,ThemeData } from '@/types';
```